modules/effects/migrations/18_0_0-beta/index.ts

```diff
@@ -107,7 +126,7 @@ export function migrateConcatLatestFromImport(): Rule {
           new InsertChange(
             sourceFile.fileName,
             effectsImportDeclaration.getEnd() + 1,
-            `${newOperatorsImport}\n`
+            `${newOperatorsImport}${os.EOL}`
```

Comment:
Thanks for the fix @rainerhahnekamp !
Could you please revert this change?
The reasoning is that we mainly use `\n` in other schematics, maybe this is incorrect but it makes snapshot testing easier 😅

Reply:
Done